### PR TITLE
Caracal container images

### PR DESCRIPTION
**Pull request by seunghun1ee**

### CODE REVIEW

**Comment by jovial** [comment]
Building octavia, neutron, nova and octavia here: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/11177744130. Reason for nova, neutron bump is to bring in: https://github.com/stackhpc/networking-mlnx/pull/4 and to use the new ubuntu base image.

**Comment by seunghun1ee** [comment]
Octavia also needs rebuild because https://github.com/stackhpc/stackhpc-kayobe-config/pull/1303 did not include Ubuntu base image bump at https://github.com/stackhpc/stackhpc-kayobe-config/pull/1312/files#diff-7d49fd564d3d78c7495b5f5c3254aaf16f453f83f1c7b80e704f304ff2ca33f7L6

**Comment by jovial** [comment]
Rebuilding neutron again for eswitchd fix: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/11371085554

**Comment by MoteHue** [comment]
Ironic bump split to a separate PR, pending testing next week: https://github.com/stackhpc/stackhpc-kayobe-config/pull/1344
@jovial This one's ready for review :)

**Comment by jovial** [comment]
Could you add a release note for the changes? 
- Base image bump for ubuntu
- https://github.com/stackhpc/stackhpc-kayobe-config/pull/1312#issuecomment-2393303675

**Comment by MoteHue** [comment]
> Could you add a release note for the changes?
> 
>     * Base image bump for ubuntu
> 
>     * [Caracal container images #1312 (comment)](https://github.com/stackhpc/stackhpc-kayobe-config/pull/1312#issuecomment-2393303675)

Added

File: releasenotes/notes/caracal-bump-container-images-083b691e83816b3b.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
```
Comment (jovial) [inline releasenotes/notes/caracal-bump-container-images-083b691e83816b3b.yaml:3]: nit: Separate into multiple list items:
```
---
features:
  - Bumped the base image for Ubuntu 22.04 containers. 
  - |
    Additionally bumped Nova, Neutron and Octavia images for
    both Rocky and Ubuntu.
```

**Review by jovial — DISMISSED** [review]
Small nit on reno, but will be good to get this in ASAP to avoid people rebuilding images and using the old ubuntu base image.

File: releasenotes/notes/caracal-bump-container-images-083b691e83816b3b.yaml (hunk 1 above)
Comment (MoteHue) [reply releasenotes/notes/caracal-bump-container-images-083b691e83816b3b.yaml:3]: Fixed

**Review by jovial — DISMISSED** [review]

**Review by Alex-Welsh — APPROVED** [review]

**Pull request merged**